### DISCUSSION TITLE
Option to serve application across local network using IPv4 Address as host

### DISCUSSION THREAD

**Discussion by ManojKiranA** [laravel/framework#39933] [Q&A] [answered]
By default while running a `serve` command it will uses the `localhost` / `127.0.0.1`. 

It would be nice to have a command to serve app using **local IPv4 Address as host** so that it can be accessed by the devices that has connected  across  the network

Eventhough `serve` command has option to pass host option it would be nice to have this command.

**Comment by cwinkler** [comment, marked as answer]
php artisan serve --host=0.0.0.0

do NOT use this in production!
